File: 241017/데이터 교환/exchange-data.c
#include <stdio.h>

int main() {
    int a=5;
    int b=6;
    int c=7;
    int temp;
    temp = a;

    a = c;
    c = b;
    b = temp;

    printf("%d\n%d\n%d",a,b,c);

    // 여기에 코드를 작성해주세요.
    return 0;
}

File: 241020/데이터 교환/exchange-data.java
public class Main {
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        int a=5;
        int b=6;
        int c=7;
        int temp;

        temp=a;
        a=c;
        c=b;
        b=temp;

        System.out.printf("%d\n%d\n%d\n",a,b,c);
    }
}